### PR TITLE
ES6 template import

### PR DESCRIPTION
**Pull request by szaliubovskii**
Controller separated to different file and add template import with es6 syntaxis.

### CODE REVIEW

File: src/app/about/components/about/about.component.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import template from './about.html';
+
 export default {
-  template: require('./about.html'),
+  template: template
```
Comment (magmel48) [inline src/app/about/components/about/about.component.js]: shorthand please there and below

**Review by magmel48 — CHANGES_REQUESTED** [review]

File: src/app/about/components/about/about.component.js (hunk 1 above)
Comment (szaliubovskii) [reply src/app/about/components/about/about.component.js]: Shorthand. Same for controller use. Fixed controller export.

**Review by magmel48 — APPROVED** [review]

**Comment by matiishyn** [comment]
Personally I don't think that having controller in a separate file is a good approach.

**Comment by szaliubovskii** [comment]
This approach was taken in basic training. Mostly people move controller to separate file and it already became guideline for angularjs development. People expect it to be in different file.

**Pull request merged**